allennlp/models/model.py

```diff
@@ -65,6 +69,18 @@ def forward(self, *inputs) -> Dict[str, torch.Tensor]:  # pylint: disable=argume
         """
         raise NotImplementedError
 
+    def get_metrics(self, reset: bool = False) -> Dict[str, float]:
+        """
+        Returns a dictionary of metrics. If the ``forward`` call of your model returns a
```

Comment:
I think this docstring is out of date.  It'd be helpful to note here (and/or in class docstring) that the model is expected to call the metrics in `forward`, with the `Metric` objects themselves handling the accumulation across calls to forward.